src/vmm_mad/remotes/lib/lxd/mapper/mapper.rb

```diff
@@ -182,15 +184,23 @@ def unmap(one_vm, disk, directory)
         OpenNebula.log_info "Unmapping disk at #{directory}"
 
         sys_parts  = lsblk('')
-        partitions = []
-        device     = ''
 
         return false if !sys_parts
 
+        partitions = []
+
+        df = "#{COMMANDS[:df]} #{directory} | grep /dev"
```

Comment:
Not sure if this work as execute works directly through popen3, there is no shell setting up the | to grep